.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Thanks for contributing to Zulip! Please follow the following pull request
+template to help maintainers review your code and merge your changes quicker.
+
+To learn more about this pull request template, read the pull request template
+guide at https://zulip.readthedocs.io/en/latest/pull-request-template.html.
+-->
```

Comment:
If this is the format we're going with, I'm not sure if we need to document the template, since this is pretty straightforward.